frontend/site/.vitepress/theme/components/InventoryTable.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
Could we make sure that the table takes 100% width even if empty?
![image](https://github.com/etalab/ouverture.data.gouv.fr/assets/28541902/5112275e-cc50-49b0-b998-fe71b7c2b1cf)

Reply:
We can force the width of the table to be fixed at 100% but it prevents the columns to adapt to their contents so I think it's worst.

## Current columns
![Xnapper-2023-06-15-09 54 55](https://github.com/etalab/ouverture.data.gouv.fr/assets/8336712/29522325-7139-4e1e-89d9-b52c9fc8eebf)

## Fixed layout
![Xnapper-2023-06-15-09 54 40](https://github.com/etalab/ouverture.data.gouv.fr/assets/8336712/42a17edc-97b1-4a0f-aac4-346873b02b77)

Reply:
We can't have an auto-layout with a min-width? :cry:

Reply:
I forgot we were in Vue, we'll do the change to fixed layout when we have no results

Reply:
done !